lib/Models/Catalog/Esri/ArcGisMapServerCatalogItem.ts

```diff
@@ -619,11 +615,11 @@ function getRectangleFromLayer(extent: Extent, rectangle: RectangleExtent) {
 
     const wkid = "EPSG:" + wkidCode;
 
-    if (!isDefined((proj4definitions as any)[wkid])) {
+    if (!isDefined((Proj4Definitions as any)[wkid])) {
```

Comment:
`(Proj4Definitions as any)[wkid]` can be simplified as `Proj4Definitions[wkid]`. The same to the rest.